src/core/store/modules/wallet/wallet.actions.ts

```diff
@@ -86,8 +87,14 @@ const walletSelect = createAsyncThunk<{ isConnected: boolean }, WalletSelectProp
               read: web3Provider.getInstanceOf(providerType),
               write: web3Provider.getInstanceOf('wallet'),
             });
+            console.log('subgraph - wallet select change network', network);
             dispatch(NetworkActions.changeNetwork({ network }));
           }
+          // Note: Force state to goerli
+          else if (networkId === 5) {
+            web3Provider.register('wallet', getEthersProvider(wallet.provider as ExternalProvider));
+            dispatch(NetworkActions.changeNetworkGoerli());
```

Comment:
can u put it inside of changeNetwork instead of a separate goerli function

Reply:
done